requirements.txt

```diff
@@ -1,3 +1,5 @@
+paddlepaddle==2.4.2
```

Comment:
for most people, they don't need paddlexxx, maybe these requirements should in other requirexxx.txt